docs-source/content/userguide/managing-domains/domain-lifecycle/liveness-readiness-probe-customization.md

```diff
@@ -0,0 +1,91 @@
+
+
+* The `spec.livenessProbeCustomScript` domain resource attribute affects all WebLogic Server instance Pods in the domain.
+* The `LIVENESS_PROBE_CUSTOM_SCRIPT` environment variable takes precedence over the `spec.livenessProbeCustomScript` domain resource attribute when both are configured, and, like all domain resource environment variables, can be customized on a per domain, per cluster, or even a per server basis.
+* Changes to either the domain resource attribute or the environment variable on a running domain take effect on already running WebLogic Server instance Pods when such Pods are restarted (rolled).
+
+**Note**: The liveness probe custom script option is for advanced usage only and its value is not set by default. If the specified script is not found, then custom script is ignored and existing liveness script will perform its normal checks.
+
+**Note**: Oracle recommends against having any long running calls (e.g. any network calls or executing wlst.sh) in the liveness probe custom script.
+
+Use the following configuration to specify liveness probe custom script using `livenessProbeCustomScript` domain resource field.
+```
+spec:
+  livenessProbeCustomScript: /u01/customLivenessProbe.sh
+```
+
+Use the following configuration to specify liveness probe custom script using `LIVENESS_PROBE_CUSTOM_SCRIPT` environment variable.
+```
+    serverPod:
+      env:
+      - name: LIVENESS_PROBE_CUSTOM_SCRIPT
+        value: /u01/customLivenessProbe.sh
+```
+
+The following environment variables that are populated by the Operator are available for use in the liveness probe custom script which will be invoked by `livenessProbe.sh`.
+
+`ORACLE_HOME` or `MW_HOME`: The Oracle Fusion Middleware software location as a file system path within the container.
+
+`WL_HOME`: The Weblogic Server installation location as a file system path within the container. 
+
+`DOMAIN_HOME`: The domain home location as a file system path within the container.
+
+`JAVA_HOME`: The Java software installation location as a file system path within the container.
+
+`DOMAIN_NAME`: The WebLogic Server domain name.
+
+`DOMAIN_UID`: The domain unique identifier.
+
+`SERVER_NAME`: The WebLogic Server instance name.
+
+`LOG_HOME`: The WebLogic log location as a file system path within the container. This variable is available only if its value is set in the configuration.
+
+**Note**: Additional environment variables that are populated by the Operator and that are not listed above are not supported for use in the liveness probe custom script; their existence or meaning may change in the future.
```

Comment:
Because you have four NOTES following each other, I would suggest that you make a bulleted list called **NOTES**: and then make each entry a sub-bulleted item in the list.

Additional environment variables that are populated by the Operator -> Additional operator-populated environment variables
and that are not listed above -> that are not listed above, (comma)
; their existence or meaning may change in the future. -> Remove this part of the sentence and end it with a period.